.github/actions/baseline-test/action.yml

```diff
@@ -0,0 +1,222 @@
+name: 'Hello World'
+description: 'Greet someone and record the time'
+inputs:
+  node-version:
+    description: 'node-version'
+    required: true
+    default: '16.x'
+  python-version: 
+    description: python-version
+    default: '3.7'
+    required: true
+  case-path: 
+    description: 'case-path'
+    required: true
+    default:  'success/case1'
+  node_env:
+    description: 'node_env'
+    required: false
+    default:  'testing'
+  jina_hubble_registry: 
```

Comment:
```suggestion
  jina-hubble-registry: 
```

Reply:
All parameters are in underlined format
avoid the middle underlining in github action